README.md

```diff
@@ -1,76 +1,126 @@
+Currently, this pipeline does not implement phasing of polyploid genomes or HiC scaffolding.
+
+## Usage
+
+> [!NOTE]
+> If you are new to Nextflow and nf-core, please refer to [this page](https://nf-co.re/docs/usage/installation) on how to set-up Nextflow. Make sure to [test your setup](https://nf-co.re/docs/usage/introduction#how-to-run-a-pipeline) with `-profile test` before running the workflow on actual data.
+
+First, prepare a samplesheet with your input data that looks as follows:
+
+`samplesheet.csv`:
+
+```csv
+sample,ontreads,hifireads,ref_fasta,ref_gff,shortread_F,shortread_R,paired
+sampleName,ontreads.fa.gz,hifireads.fa.gz,assembly.fasta.gz,reference.fasta,reference.gff,short_F1.fastq,short_F2.fastq,true
+```
+
+Each row represents one genome to be assembled. `sample` should contain the name of the sample, `ontreads` should contain a path to ONT reads (fastq.gz), `hifireads` a path to HiFi reads (fastq.gz), `ref_fasta` and `ref_gff` contain reference genome fasta and annotations. `shortread_F` and `shortread_R` contain paths to short-read data, `paired` indicates if short-reads are paired. Columns can be omitted if they contain no data, with the exception of `shortread_R`, which needs to be present if `shortread_F` is there, even if it is empty.
+
+-->
+
+Now, you can run the pipeline using:
+
+```bash
+nextflow run nf-core/genomeassembler \
+   -profile <docker/singularity/.../institute> \
+   --input samplesheet.csv \
+   --outdir <OUTDIR>
+```
+
+> [!WARNING]
+> Please provide pipeline parameters via the CLI or Nextflow `-params-file` option. Custom config files including those provided by the `-c` Nextflow option can be used to provide any configuration _**except for parameters**_; see [docs](https://nf-co.re/docs/usage/getting_started/configuration#custom-configuration-files).
+
+For more details and further functionality, please refer to the [usage documentation](https://nf-co.re/genomeassembler/usage) and the [parameter documentation](https://nf-co.re/genomeassembler/parameters).
+
+### Pre-set profiles
+
+To ease configuration, there are a couple of pre-defined profiles for various combinations of read sources and assemblers (named readtype_assembler)
+
+| ONT | HiFI  | Assembly-strategy                                  | Profile name          |
+| --- | ----- | -------------------------------------------------- | --------------------- |
+| Yes | No    | flye                                               | `ont_flye`            |
+| No  | Yes   | flye                                               | `hifi_flye`           |
+| No  | Yes   | hifiasm                                            | `hifi_hifiasm`        |
+| Yes | Yes   | hifiasm --ul                                       | `hifiont_hifiasm`     |
+| Yes | Yes   | Scaffolding of ONT assemblies onto HiFi assemblies | `hifiont_flyehifiasm` |
+
+### Pipeline specific parameters
```

Comment:
This whole section isnt supposed to be here in the `README.md` (I would vote for deleting it) but rather in `nextflow_schema.json` (amend if neccessary, but there are some entries already, didnt check for completeness) so that it will be presented in https://nf-co.re/genomeassembler/parameters/.